java/src/jmri/jmrix/cmri/serial/sim/configurexml/ConnectionConfigXml.java

```diff
@@ -33,6 +33,11 @@ protected void getInstance() {
     }
 
     @Override
+    protected void getInstance(Object object) {
```

Comment:
Is there a reason you used an Object and not a ConnectionConfig as the parameter?